src/main/java/com/iota/iri/Snapshot.java

```diff
@@ -108,7 +114,7 @@ public int index() {
         return i;
     }
 
-    private Snapshot(Map<Hash, Long> initialState, int index) {
+    public Snapshot(Map<Hash, Long> initialState, int index) {
```

Comment:
since we use the index of the snapshots as the real "latestSolidMilestoneIndex" rather than a copied value to avoid race conditions and to prepare for local snapshots, we need to create Snapshots in some tests manually (see EntryPointSelectorImplTest.java) for mocking